packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -442,6 +442,28 @@ class FuchsiaDevice extends Device {
     }
   }
 
+  @override
+  bool get supportsScreenshot => true;
+
+  @override
+  Future<void> takeScreenshot(File outputFile) async {
+    if (outputFile.basename.split('.').last != 'ppm') {
```

Comment:
Not super familiar with this flow, but it looks like you can set the file output name in the `flutter screenshot` command, but `ResidentRunner.screenshot()` hardcodes 'flutter.png'.

Reply:
If it sounds reasonable, I can follow up with adding a `defaultScreenshotName` to `Device` and then let ResidentRunner.screenshot() read the value.

Reply:
sg. it looks like the resident runner screenshot call is already using `getUniqueFile`, so `Device` might need both a `defaultScreenshotName` and a `defaultScreenshotExt`.